types/bull-arena/package.json

```diff
@@ -1,6 +1,7 @@
 {
     "private": true,
     "dependencies": {
+        "@types/ioredis": "^4.28.10",
```

Comment:
bull-area appears to be unmaintained (per a pinned issue on their repo); I'm not going to attempt to figure out how to make this work with ioredis 5.

Reply:
this is perfectly fine approach here IMO